test/common/bond_test.js

```diff
@@ -68,20 +68,38 @@ test('determineBondProvider should return the AnVIL BondProvider if the URL host
     t.is(determineBondProvider('drs://dg.ANV0/00008531-03d7-418c-b3d3-b7b22b5381a0'), BondProviders.ANVIL);
 });
 
-test('determineBondProvider should return the AnVIL BondProvider if the URL host is the AnVIL host', (t) => {
-    t.is(determineBondProvider('drs://gen3.theanvil.io/dg.ANV0/00008531-03d7-418c-b3d3-b7b22b5381a0'), BondProviders.ANVIL);
+test('determineBondProvider should return the AnVIL BondProvider if the URL host is the AnVIL prod host', (t) => {
+    t.is(determineBondProvider(`drs://${config.HOST_THE_ANVIL_PROD}/dg.ANV0/00008531-03d7-418c-b3d3-b7b22b5381a0`), BondProviders.ANVIL);
 });
 
-test('determineBondProvider should return the dcf-fence BondProvider if the URL host is Kids First', (t) => {
+test('determineBondProvider should return the AnVIL BondProvider if the URL host is the AnVIL staging host', (t) => {
+    t.is(determineBondProvider(`drs://${config.HOST_THE_ANVIL_STAGING}/dg.ANV0/00008531-03d7-418c-b3d3-b7b22b5381a0`), BondProviders.ANVIL);
+});
+
+test('determineBondProvider should return the dcf-fence BondProvider if the URL host is Kids First prod', (t) => {
+    t.is(
+        determineBondProvider(`drs://${config.HOST_KIDS_FIRST_PROD}/ed6be7ab-068e-46c8-824a-f39cfbb885cc`),
+        BondProviders.DCF_FENCE,
```

Comment:
dcf-fence is for CRDC. Is it also used for Kids First? Or are we just asserting the current behavior even if it's not confirmed to be correct? If that's the case, we should put a comment on these tests.

Reply:
[uhh...](https://github.com/broadinstitute/martha/blob/e1039568bf81c2b7d952d9881c2a9731d954a5ab/common/bond.js#L36)

Reply:
Actually that link I posted above is a red herring as the `config.HOST_KIDS_FIRST_PROD` URL under test is not a dg URL. `config.HOST_KIDS_FIRST_PROD` [falls all the way through](https://github.com/broadinstitute/martha/blob/e1039568bf81c2b7d952d9881c2a9731d954a5ab/common/bond.js#L59) to the bottom and gets the `default` [DCF_FENCE bond provider](https://github.com/broadinstitute/martha/blob/e1039568bf81c2b7d952d9881c2a9731d954a5ab/common/bond.js#L11).